conf/ConfigureFromEnv.php

```diff
@@ -139,7 +142,12 @@
 }
 
 if(defined('SS_ERROR_LOG')) {
-	SS_Log::add_writer(new SS_LogFileWriter(BASE_PATH . '/' . SS_ERROR_LOG), SS_Log::WARN, '<=');
+	$logger = Injector::inst()->get('Logger');
+	if($logger instanceof Logger) {
+		$logger->pushHandler(new StreamHandler(BASE_PATH . '/' . SS_ERROR_LOG, Logger::WARN));
```

Comment:
Rather than adding this handler here, why not create a new custom SS handler (which extends StreamHandler) and then add this to the logger via yml.

You can add the logic for `if(defined('SS_ERROR_LOG')` to the class itself, which does nothing if it's not defined.

Less coupling between monolog that way. :)